.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+src/main/webapp/WEB-INF/views/work/*
```

Comment:
What's there?

Reply:
Just folder for collection of .jsp files from the web.